Shared/Extensions/NSFileManagerExtensions.swift

```diff
@@ -10,60 +10,60 @@ import Foundation
 public let NSFileManagerExtensionsDomain = "org.mozilla.NSFileManagerExtensions"
 
 public enum NSFileManagerExtensionsErrorCodes: Int {
-    case EnumeratorFailure = 0
-    case EnumeratorElementNotURL = 1
-    case ErrorEnumeratingDirectory = 2
+    case enumeratorFailure = 0
+    case enumeratorElementNotURL = 1
+    case errorEnumeratingDirectory = 2
 }
 
-public extension NSFileManager {
+public extension FileManager {
 
-    private func directoryEnumeratorForURL(url: NSURL) throws -> NSDirectoryEnumerator {
+    fileprivate func directoryEnumeratorForURL(_ url: URL) throws -> FileManager.DirectoryEnumerator {
```

Comment:
I dont know what we wanna do with cases like this. Should we rename it correctly or just aim to get it work for now. `directoryEnumeratorForURL(_ url: URL)` could be `directoryEnumerator(forURL url: URL)`